assets/src/dashboard/components/cardGallery/components.js

```diff
@@ -97,7 +90,9 @@ export const MiniCard = styled.div(
     position: relative;
     width: ${width}px;
     cursor: pointer;
-    border: ${theme.DEPRECATED_THEME.borders.gray75};
+    border: 1px solid ${theme.colors.border.defaultNormal};
+    border-radius: ${theme.borders.radius.small};
+ 
```

Comment:
Are the mini cards supposed to be updated here too? It looks like they need to be a little bigger

|Current|Designs|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/110033139-3a7cf980-7cf6-11eb-91aa-11a34085ca28.png)|![image](https://user-images.githubusercontent.com/22185279/110033081-26d19300-7cf6-11eb-8969-a00a2b13fdc2.png)|

Reply:
this is a tricky one. The templates and stories have to be kept at the right ratio of a story (this util: `getPagePreviewHeights  ... Math.round((width / PAGE_RATIO) * 100) / 100;` ) so this gallery updates responsively according to how much room there is. 

I would rather handle any necessary updates to this as its own thing? 😬 


https://user-images.githubusercontent.com/10720454/110037252-60f16380-7cfb-11eb-951f-b7aa3e9d19b8.mp4